apps/condo/domains/banking/access/BankCategory.js

```diff
@@ -5,18 +5,26 @@
 
 const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
 
-async function canReadBankCategories ({ authentication: { item: user } }) {
+const { checkBankIntegrationsAccessRights } = require('@condo/domains/banking/utils/accessSchema')
+
+const { BANK_INTEGRATION_IDS } = require('../constants')
+
+async function canReadBankCategories ({ authentication: { item: user }, context }) {
     if (!user) return throwAuthenticationError()
     if (user.deletedAt) return false
 
+    if (await checkBankIntegrationsAccessRights(context, user.id, [BANK_INTEGRATION_IDS.SBBOL])) return true
+
     return {}
```

Comment:
why you check checkBankIntegrationsAccessRights if in the end you return true
{} - means no modifications on query to get objects